operator/v1/types.go

```diff
@@ -55,7 +55,19 @@ type ClusterManagerSpec struct {
 	DeployOption ClusterManagerDeployOption `json:"deployOption,omitempty"`
 }
 
-// DetachedClusterManagerConfiguration represents customized configurations we need to set for clustermanager in the detached mode.
+// HostedClusterManagerConfiguration represents customized configurations we need to set for clustermanager in the Hosted mode.
+type HostedClusterManagerConfiguration struct {
+	// RegistrationWebhookConfiguration represents the customized webhook-server configuration of registration.
+	// +optional
+	RegistrationWebhookConfiguration WebhookConfiguration `json:"registrationWebhookConfiguration,omitempty"`
+
+	// WorkWebhookConfiguration represents the customized webhook-server configuration of work.
+	// +optional
+	WorkWebhookConfiguration WebhookConfiguration `json:"workWebhookConfiguration,omitempty"`
+}
+
+// DetachedClusterManagerConfiguration represents customized configurations we need to set for clustermanager in the Hosted mode.
```

Comment:
I think it is save to remove this part. I do not think we have another consumer in old release that uses this field

Reply:
Sure. let me remove it.